File: src/Deviser.Core/Deviser.Core.Data/Entities/Module.cs
using System;
using System.Collections.Generic;

namespace Deviser.Core.Data.Entities
{
    public partial class Module
    {
        public Module()
        {
            ModuleView = new HashSet<ModuleView>();
            PageModule = new HashSet<PageModule>();
        }

        public Guid Id { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
        public string Label { get; set; }
        public DateTime? LastModifiedDate { get; set; }
        public string Name { get; set; }
        public string Version { get; set; }
        public virtual ICollection<ModuleView> ModuleView { get; set; }
        public virtual ICollection<PageModule> PageModule { get; set; }
        public virtual ICollection<AdminPage> AdminPage { get; set; }

    }
}

File: src/Deviser.Core/Deviser.NgLib/deviser-workspace/projects/deviser-shared/src/lib/domain-types/module.ts
import { ModuleView } from './module-view';

export interface Module {
    id: string;
    createdDate: Date | string | null;
    description: string;
    isActive: boolean;
    label: string;
    lastModifiedDate: Date | string | null;
    name: string;
    version: string;
    moduleView: ModuleView;
    //IsActiveText: string;
    isActiveBadgeClass: string;
}